### ISSUE TITLE
Your computer or network may be sending automated queries. To protect our users, we can't process your request right now.

### ISSUE THREAD

**Issue by gabrieligbastos** [sarperavci/GoogleRecaptchaBypass#3]
Hi,

First of all, congratulations on the project! :)

So, developed here a test, of a hourly crawler...
![image](https://github.com/sarperavci/GoogleRecaptchaBypass/assets/2159153/752c05a0-bf76-44a7-ad0e-7c645231aad7)

Have you already got that?
Any suggestion to avoid getting this?

Is there any way to reset cache? Im thinking weird, because this is happening on my Chrome.
But if i try on Brave manually, it is working!

Thanks you,

**Comment by sarperavci** [comment]
Look at the main page:

**One warning: Google may block your IP if you solve too many captchas in a short period of time. So, use this script wisely or change your IP frequently.**

That problem is inevitable when you solve many reCAPTCHAs. So, you should change your IP address. 

I'll add Selenium implementation soon for the RecaptchaSolver class. You could try it with Brave Browser webdriver.

**Comment by gabrieligbastos** [comment]
No problem, i didnt think hourly were short period of time. Do you have any notion on a more likely safe-range?

And the Brave, was that i thought it was strange, if it was an IP related issue.

**Comment by sharpprahs** [comment]
@gabrieligbastos were you able to find the optimal time frame?

**Comment by gabrieligbastos** [comment]
No, it was for a hobby project, just forgot about it. If you find out, let me know! :)

**Comment by sarperavci** [comment]
Rotating IP is the most reliable way
